pymesync.py

```diff
@@ -133,6 +134,15 @@ def create_time(self, parameter_dict):
         ``parameter_dict`` is a python dictionary containing the time
         information to send to TimeSync.
         """
+        if type(parameter_dict['duration']) is not int:
+            duration = self.__interpret_times(parameter_dict['duration'])
+            parameter_dict['duration'] = duration
+
+            if parameter_dict['duration'] is None:
+                error_msg = [{self.error:
+                              "time object: duration contains invalid string"}]
```

Comment:
Actually, thinking about it, I think this should return "invalid duration string" but make sure that an integer value is also tested and returns "invalid duration integer"